packages/jsii-calc/test/assembly.jsii

```diff
@@ -761,7 +761,7 @@
         {
           "docs": {
             "stability": "stable",
-            "summary": "(deprecated) String representation of the value."
```

Comment:
So many trees are going to be saved!

Reply:
Could be cool to calculate the negative carbon footprint of this change

Reply:
I agree but don't you try to nerd snipe me into sinking days or weeks into this. I'd first need to have representative telemetry for "cumulative compile times" + roughly what kind of hardware is in use (to estimate average power consumption of that build), and a ballpark idea of where that is geographically (to estimate the carbon intensity of the power supply there).

But I'd lie if your initial reaction to this did not make me think "oh jeez, it'd be sweet if the PRs had an automated comment telling you what the impact of the change is on the carbon footprint of the codebase"...